### DISCUSSION TITLE
Release cadence?

### DISCUSSION THREAD

**Discussion by mdo** [twbs/bootstrap#30602] [General]
Trying something new!

How frequently would you prefer Bootstrap releases (minor and patch) to occur? **Vote with reactions to this comment please**, and feel free to expand below.

- 😃 Whenever
- 🚀 Once every 6-8 weeks
- ❤️ Once every 4 weeks
- 🎉 Once every two weeks

We're able to automate enough things about our releases, but we tend to hold onto them for other reasons and let them run long. Hoping to gather some community feedback and see if we can have a stronger, more expected release cycle.

**Comment by mdo** [comment]
We've just released v4.5.0 and have already scheduled v4.5.1 for a month out. We'll let that release happen with whatever comes naturally there as we focus on v5's first alpha :).

**Comment by ascensus-mdb** [comment]
Hi. What happened to this idea? I really hoped in V5 you will keep releasing every 4 weeks, as the community voted.

**Reply by patrickhlauke** [reply]
Hmm, I wonder what might have happened globally over the last few months that may have slowed down/delayed activity...

**Reply by ascensus-mdb** [reply]
Just asking, what is this irony for? I see a lot of news in recent times about new bootstrap t-shirts and icons, so I dared to ask what happened with bootstrap itself.
https://twitter.com/getbootstrap/status/1283766350658383872

**Comment by XhmikosR** [comment]
Real life™ happened, but we'll get back to it.

**Comment by 2acode** [comment]
Hello,
I'm reopening this old thread for a clarification (and suggestion) regarding the release cycle.
I state that it is not a controversy, I will never stop thanking you for the incredible work you are doing.

In my opinion it's more useful if you release closer together with fewer changes.
For example now, for over 1 month, there are over 200 tasks completed in the alpha version and everything seems to be "stuck".

I don't mean a fixed release every certain time, but a release when there are a certain amount of tasks completed (20, 30, 40 depending on the type and importance).
For example, let's say that among the tasks there are two new ready-made features that I would like to use but I have to wait for the completion of another 200 tasks and 1 or 2 months before I can use it in a stable release.

This allows you to better focus on the new features released, to use them immediately as soon as they are ready and to give the project a "higher speed" feeling.

Thank you
Alessandro

**Reply by mdo** [reply]
Agreed. I've been traveling so it's been hard to stick to a timeline. We'll be shipping the next alpha hopefully next week when I'm back.
